src/css/profile/mobile/common/datepicker.less

```diff
@@ -0,0 +1,417 @@
+.tau_datepicker_material {
+	position: absolute;
+	display: block;
+	background-color: white;
+	font-size: 14px;
```

Comment:
Use the '14 * @px_base' for modifiability.

Reply:
done.